### ISSUE TITLE
Evaluation and Testing

### ISSUE THREAD

**Issue by SnowyPlums** [SnowyPlums/ML-Streamlit#4]
Evaluate model performance: Calculate accuracy, precision, recall, F1 score for ANN and other models.

Generate confusion matrix: Visualize model predictions.

Compare model performances: Compare metrics between ANN and other models.

Plot ROC curve and AUC score: Analyze model performance visually.

Fine-tune hyperparameters for ANN: Adjust parameters like learning rate, epochs, batch size for ANN.
^^Will be very time consuming
